Automattic/wp-calypso
# Remove the breadcrumb if jetpack in campaign list (blazepress)

Related to #

## Proposed Changes

* Remove the breadcrumb and back button in BlazePress campaign details if the app is Jetpack

## Testing Instructions
* Open a campaign in calypso (no jetpack). It should display the breadcrumb. If you put the browser in mobile it will display the "back" button
* Open a campaign in Jetpack. It should not display the breadcrumb and the back button

## Pre-merge Checklist
